### PR TITLE
Added Alteration to Characters

### PR DESCRIPTION
**Pull request by JFranklin3000**
Closes [issue URL]

##### In this PR
- Created Alteration Item
- Added Alterations to the Gear Tab

##### Testing
- Create Item and Drag and Drop to Actor sheet.

### CODE REVIEW

File: templates/actor/parts/items/alteration/details.hbs
Comment (phildominguez) [inline templates/actor/parts/items/alteration/details.hbs:1]: Would chips for the essences/costs be a good idea?

File: lang/en.json [hunk 1]
```diff
@@ -75,6 +75,11 @@
     "ActorSizeTitanic": "Titanic",
     "ActorSizeTowering": "Towering",
     "ActorStun": "Stun",
+    "Alteration": "Alteration",
+    "AlterationBonus": "Bonus",
+    "AlterationCost": "Cost",
```
Comment (phildominguez) [inline lang/en.json:80]: We already have `E20.Bonus`, and we have a few entries that are just "Cost" as well. Would be good to just have `E20.Cost` I think.

File: template.json [hunk 2]
```diff
@@ -635,6 +636,15 @@
         }
       }
     },
+    "alteration": {
+      "templates": [
+        "itemDescription"
+      ],
+      "benefit": "",
+      "cost": "",
```
Comment (phildominguez) [inline template.json:644]: Aren't these numbers?

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: lang/en.json (hunk 1 above)
Comment (JFranklin3000) [reply lang/en.json:80]: That makes sense to go to one name.

File: template.json (hunk 2 above)
Comment (JFranklin3000) [reply template.json:644]: Nope benefit can be other things. Basically the 3 main things that are a benefit/ cost are essences, movement, and Other things. Here is a good example. 

image

File: templates/actor/parts/items/alteration/details.hbs
Comment (JFranklin3000) [reply templates/actor/parts/items/alteration/details.hbs:1]: yea I need to figure out the best layout for the various types yet.

**Review by phildominguez — APPROVED** [review]

**Pull request merged**